packages/web-react/src/components/TooltipModern/README.md

```diff
@@ -0,0 +1,9 @@
+# TooltipModern
+
+⚠️ `TooltipModern` component is [deprecated] and will be renamed to `Tooltip` in the next major version.
+
+For more information and implementation examples, please visit the [`TooltipModern` section][tooltip-modern-section] section in the [readme][tooltip] for the `Tooltip` component.
+
+[deprecated]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web-react/README.md#deprecations
```

Comment:
this is not used

Reply:
it was there, just fixed the markdown link to follow all other links `[deprecated]` -> `[deprecated][deprecated]`